tfhe-zk-pok/src/proofs/pke.rs

```diff
@@ -192,6 +195,7 @@ pub fn prove<G: Curve>(
 
     assert!(c2.len() <= k);
     let k = k.min(c2.len());
+    let (_, big_d, _) = compute_crs_params(d, k, b_i, q, t);
```

Comment:
why this change ? 🤔

Reply:
i think i was experimenting with using a different value of D when using a smaller `k`, but forgot to remove it. would you like me to undo it?

Reply:
if the change is not required yes please